frontend/libs/studio-components/README.md

```diff
@@ -0,0 +1,13 @@
+# @altinn/studio-components
+
+Studio-Components is a tools that wraps around `@digdir/design-system-react` and `@digdir/design-system-tokens`, giving you access to the Designsystem's componetns and tokens adding Altinn customisation to them. It's great for delvelopers and designers to have this library for components to ensure that the components used throughout altinn studio's applications looks great and stay consistent.
```

Comment:
Det er litt mer enn bare en fasade og wrapper rundt tredje komponenter. Det er også interne komponenter for Studio som skal følge et "dummy-components" pattern. 😊